code/modules/paperwork/records/info/zayin.dm

```diff
@@ -61,3 +61,11 @@
 		"The employee that worked on F-04-36 recovered small amounts of HP and SP over time.",
 		"When the work result was bad, the employee working on F-04-36 decided to sit down for a break.",
 		"Agent Bong bong had to be forcibly removed after being caught sitting on F-04-36 for an entire shift.")
+
+//Quiet Day
+/obj/item/paper/fluff/info/zayin/quiet
+	abno_type = /mob/living/simple_animal/hostile/abnormality/quiet_day
+	abno_code = "T-01-144"
```

Comment:
Theonite slab was recently merged with this abno code, it will need to be changed. 145 should be fine.